light-client-verifier/src/predicates.rs

```diff
@@ -185,6 +185,23 @@ pub trait VerificationPredicates: Send + Sync {
         Ok(())
     }
 
+    /// Check that there is enough signers overlap between the given, untrusted validator set
+    /// and the untrusted signed header based on the specified trust threshold.
+    fn has_specified_signers_overlap(
+        &self,
+        untrusted_sh: &SignedHeader,
+        untrusted_validators: &ValidatorSet,
+        calculator: &dyn VotingPowerCalculator,
+        trust_threshold: TrustThreshold,
+    ) -> Result<(), VerificationError> {
+        calculator.check_signers_overlap_level(
```

Comment:
And likewise here...

Reply:
01c5112f3625905bb00db13c030254532410adf4